pkg/cmd/image/inspect.go

```diff
@@ -22,31 +22,14 @@ import (
 	"time"
 
 	"github.com/containerd/containerd"
-	"github.com/containerd/containerd/platforms"
 	"github.com/containerd/nerdctl/pkg/api/types"
-	"github.com/containerd/nerdctl/pkg/clientutil"
 	"github.com/containerd/nerdctl/pkg/formatter"
 	"github.com/containerd/nerdctl/pkg/idutil/imagewalker"
 	"github.com/containerd/nerdctl/pkg/imageinspector"
 	"github.com/containerd/nerdctl/pkg/inspecttypes/dockercompat"
 )
 
-func Inspect(ctx context.Context, imageFilter []string, options types.ImageInspectOptions) error {
-	var clientOpts []containerd.ClientOpt
-	if options.Platform != "" {
-		platformParsed, err := platforms.Parse(options.Platform)
-		if err != nil {
-			return err
-		}
-		platformM := platforms.Only(platformParsed)
-		clientOpts = append(clientOpts, containerd.WithDefaultPlatform(platformM))
-	}
-	client, ctx, cancel, err := clientutil.NewClient(ctx, options.GOptions.Namespace, options.GOptions.Address, clientOpts...)
-	if err != nil {
-		return err
-	}
-	defer cancel()
-
```

Comment:
This code is indeed duplicated, but can you call `NewClientWithPlatform` for now instead of creating a `clientutil.options.go`.

Reply:
https://github.com/containerd/nerdctl/blob/67a065a80fceb7602a9092dc5dec52a959454853/pkg/clientutil/client.go#L59